app/core/Router.coffee

```diff
@@ -1,6 +1,28 @@
 locale = require 'locale/locale'
+normalizeViewRoute = (path) -> if not _.string.startsWith(path, 'views/') then "views/#{path}" else path
 
-go = (path, options) -> -> @routeDirectly path, arguments, options
+go = ({path, load}, options) ->
+  normedPath = normalizeViewRoute(path)
+  f = -> @routeDirectly path, arguments, options
```

Comment:
Why not pass in `normedPath` instead of `path` to `routeDirectly`?  It looks like we're normalizing the path downstream anyway, so let's do that in fewer places if possible.